packages/agoric-cli/src/lib/rpc.js

```diff
@@ -143,7 +147,11 @@ export const makeVStorage = (powers, config = networkConfig) => {
           ));
           // console.debug('readAt returned', { blockHeight });
         } catch (err) {
-          if (err.message.match(/unknown request/)) {
+          if (
+            err.message.match(/unknown request/) ||
```

Comment:
Any reason to keep this check around or can we remove it?

Reply:
to not break mainnet and devnet users

Reply:
Ah I just tested on mainnet, I'm getting error code "6", so looks like the code changed and not just the message

Reply:
Yup: https://github.com/Agoric/agoric-sdk/commit/dafc7c1708977aaa55e245dc09a73859cf1df192#diff-5ddf33f542fbc7bc5cec3cef81c28de46085850594ce2f5c324115b88cd8309aR59